### PR TITLE
signal dashboard

### PR DESCRIPTION
**Pull request by sgratzl**
WIP PR for current state of signal-dashboard -> https://deploy-preview-849--cmu-delphi-covidcast.netlify.app/indicator-status

### CODE REVIEW

**Comment by sgratzl** [comment]
current state (with the given mock data):

image

**Comment by sgratzl** [comment]
current state (with the limited live data):

image

image

**Comment by Letsgohyde** [comment]
Hey @sgratzl - I guess you did have a tracking bug for this, adding the one i just created: https://github.com/cmu-delphi/www-covidcast/issues/859

Maybe we should merge them?

**Comment by sgratzl** [comment]
@tildechris  do we have a consensus on the final design?

**Comment by tildechris** [comment]
What we have here in this PR looks good.  We can do a design review after we get some real data from the API.  In the meantime, I'll comment on the design in #859.

File: src/modes/survey/KPI.svelte [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<script>
```
Comment (tildechris) [inline src/modes/survey/KPI.svelte:1]: I didn't expect to see survey related changes here.  Can you comment on their purpose?

Comment (sgratzl) [reply src/modes/survey/KPI.svelte:1]: refactoring to the key card style such that I can also use it to show dates instead of just numbers

**Review by tildechris — APPROVED** [review]
LGTM.

**Pull request merged**